_pages/typecheck.md

```diff
@@ -684,3 +684,358 @@ Cyclic module dependencies can cause problems for the type checker.  In order to
 ```lua
 local myModule = require(MyModule) :: any
 ```
+
+## Type functions
+
+Type functions are functions that operate on types instead of runtime values. They can use the [types](#types-library) library to transform existing types or create new ones.
+
+Many common type functions are available through the community [TypeForge](https://github.com/cameronpcampbell/typeforge) library.
```

Comment:
I don't think we should be blessing Cameron's TypeForge library, even if it is a nice thing to exist. We don't want to unnecessarily favor first-arrivals.

```suggestion
```